.github/workflows/check-release.yml

```diff
@@ -0,0 +1,81 @@
+name: Check versions and build-publish
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  check-version:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python
+      uses: actions/setup-python@v3
+      with:
+        python-version: 3.8
```

Comment:
Can we also use a newer python version? I'd go for the most recent Kedro supports (so 3.10) if possible.